apps/hash-ai-worker-ts/src/activities/flow-activities/research-entities-action/infer-entities-from-web-page-worker-agent.ts

```diff
@@ -664,16 +699,34 @@ export const inferEntitiesFromWebPageWorkerAgent = async (params: {
               };
             }
 
+            if (hmtlContent.endsWith("...")) {
+              return {
+                ...toolCall,
+                output: dedent(`
+                  You provided a truncated text from the web page. You must provide the entire
+                  text necessary to accurately infer properties of the relevant entities.
+
+                  You must make the "inferEntitiesFromWebPage" tool call again with the full text
+                  required to infer the expected number of entities.
```

Comment:
We make a few references to `text` in these and other messages, it might be worth seeing which should be replaced with 'HTML'

Reply:
Good spot, have addressed in https://github.com/hashintel/hash/pull/4305/commits/ceaae237ecb58166cfc9d4ab324d0f6ce18cbfd5